config/routes.rb

```diff
@@ -60,6 +60,7 @@
   resources :admin, only: [:index]
   get 'admin/users', to: 'admin#users'
   get 'admin/counts', to: 'admin#counts'
+  get 'admin/days_since_last_login_for_inactivity', to: 'admin#days_since_last_login_for_inactivity', as: :days_since_last_login_for_inactivity
```

Comment:
Rather than making a separate endpoint for this, I would suggest just including the value in the response to the request for the admin page here:
https://github.com/SaraAlert/SaraAlert/blob/0cc8b92251b8bc8ac8e5d0466745182ffbb5e21d/app/views/admin/index.html.erb#L2-L5

And then the `AdminTable` component can just pass it along to the `UserModal`

Reply:
I agree with this, then you can ignore this [comment](https://github.com/SaraAlert/SaraAlert/pull/1180#discussion_r784857637) since it will be coming from an erb file

Reply:
[15b93cb85d43b65216cde511f24c323941562ae6](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/15b93cb85d43b65216cde511f24c323941562ae6)